pkg/invariants/kubeapiserver/disruptionnewapiserver/invariant.go

```diff
@@ -50,9 +49,5 @@ func (w *newAPIServerDisruptionChecker) WriteContentToStorage(ctx context.Contex
 }
 
 func (w *newAPIServerDisruptionChecker) Cleanup(ctx context.Context) error {
-	if err := sampler.TearDownInClusterMonitors(w.adminRESTConfig); err != nil {
```

Comment:
What deletes the namespace with the daemonset now?

Reply:
Do you wan tto remove this entire file?

Reply:
InvariantInClusterDisruption.Cleanup [deletes](https://github.com/openshift/origin/pull/28081/files#diff-6cd7ec3d1d5939351be6f7e0e42b2ad9f12c2c90806323e9245ed89ecc91b043R442-R446) namespace and then removes [CRBs and roles](https://github.com/openshift/origin/pull/28081/files#diff-6cd7ec3d1d5939351be6f7e0e42b2ad9f12c2c90806323e9245ed89ecc91b043R448-R463)